### DISCUSSION TITLE
Trying to hide the Logger Bar in the console

### DISCUSSION THREAD

**Discussion by Ernesto-Alves67** [Zulko/moviepy#2145] [Q&A] [answered]
# Recently i made a YouTube Downloader, and add GUI to it.

> - Using the **anaconda** enviroment to run and build.
> - The problem is when i build the app with pyinstaller.
> - I'm using this comand to build

```
pyinstaller  --hidden-import=PyQt5 --hidden-import=threading --hidden-import=time 
--hidden-import=pytube --hidden-import=moviepy --hidden-import=moviepy.editor --hidden-import=os 
--hidden-import=json --hidden-import=sys --hidden-import=youtubesearchpython --hidden-import=pygame

```

> - I try to change the files **AudioClip.py** and **ffmpeg_audiowriter.py** in an attempt to disable the logger bar.
> - And that's works only when i running the app using the following comand:

```
python ytdl_main.py

```
> The app works normaly and do not print the audio writting process.
>> On the main code I also direct the outputs of the application to itself with:

```Python

    def initUI(self):
        self.setWindowIcon(QtGui.QIcon(self.ui.iconspath+'app-64.png'))
        sys.stdout = self

```

# My question is:

> Why when I compile with pyinstaller, without  '**_--noconsole_**' tag to hide the console, I run the .exe, and when writing the audio, it still writes progress to the console?
>> I have done tests with '**_--noconcole_** ' too. But when he goes to write audio he can't finish the process. 

**Project Link:** https://github.com/Ernesto-Alves67/YoutubeDownloader_/tree/main/Gui_version

**Comment by hetari** [comment, marked as answer]
try this:
```py
clip = VideoFileClip("video.mp4") # Generate a clip 
clip.write_videofile("output.mp4") # Prints progress bar and info 
clip.write_videofile("output.mp4", verbose=False) # Just prints progress bar 
clip.write_videofile("output.mp4", verbose=False, progress_bar=False) # Prints nothing 
```
or try this instead `logger=None`

**Reply by Ernesto-Alves67** [reply]
Thank you very much. It helped a lot indeed. I was able to do what I wanted.  I hadn't thought about these parameters before, I was forgetting about them.

**Comment by abhaybd** [comment]
The accepted answer is no longer accurate. Setting `logger=None` is now the right way to disable logs.

**Reply by iperera97** [reply]
this is the right way to do this.
